src/components/CommunitiesSelector/CommunitiesSelector.js

```diff
@@ -33,19 +33,25 @@ export default class CommunitiesSelector extends Component {
 
   findSuggestions = (searchText) => {
     const { options, selected } = this.props
-    const newSuggestions = differenceBy(options, selected, 'id')
-      .filter(o => o.name.match(new RegExp(searchText, 'i')))
-    this.setState({ suggestions: newSuggestions })
+    let newSuggestions
+    if (searchText && searchText > 0) {
```

Comment:
did you mean searchText.length? i could also see calling .trim on the searchText to ignore whitespace

Reply:
yes, sounds good 👍